crates/config/src/lib.rs

```diff
@@ -248,6 +248,15 @@ pub struct Config {
     pub eth_rpc_jwt: Option<String>,
     /// Timeout that should be used for any rpc calls
     pub eth_rpc_timeout: Option<u64>,
+    /// Headers that should be used for any rpc calls
+    ///
+    /// # Example
+    ///
+    /// rpc_headers = ["x-custom-header:value", "x-another-header:another-value"]
+    ///
+    /// You can also the ETH_RPC_HEADERS env variable like so:
+    /// `ETH_RPC_HEADERS="x-custom-header:value x-another-header:another-value"`
+    pub rpc_headers: Option<Vec<String>>,
```

Comment:
for consistency should this field be named `eth_rpc_headers`?

Reply:
Fixed